CHANGELOG.md

```diff
@@ -121,6 +121,10 @@ v0.27
 * We now read the effective user's configuration file instead of the real user's
   configuration in case libgit2 runs as part of a setuid binary.
 
+* The `git_odb_open_rstream` and its companion in the `git_odb_backend`
+  interface has changed its signature to allow providing its size and type to
```

Comment:
"its" should probably be "the object's", to make that a bit more clear. I'd also mention the companion directly, as otherwise one has to guess what "its companion" refers to